features.md

```diff
@@ -126,3 +126,50 @@ clangd respects your project's `.clang-format` file which controls styling
 options.
 
 (Format-as-you-type is experimental and doesn't work well yet).
+
+
+## Refactoring
+
+### Rename
+
+Rename a symbol under the cursor. All usages of the symbol will be renamed,
+including **declaration**, **definition** and **references**.
+
+![screenshot: rename](screenshots/rename.gif)
+
+Most symbols are renameable, such as classes, variables, functions and methods.
+
+**KNOWN LIMITATIONS**
```

Comment:
I think this is too much info in too terse a style compared to the way other features are described, and is version-specific and will quickly go out of date. In general this isn't a place to list bugs, but describing the scope of the feature seems OK.

I'd suggest something like:
- References in templates and macro bodies may not be renamed (these are difficult to analyze in general).
- References in comments and disabled preprocessor sections are not yet renamed.
- Related symbols (like overridden methods in a class hierarchy) are not yet included.

This implicitly covers template specializations (they're related symbols), but drops:
 - exactly which weird symbols aren't supported (this is a mix of things people won't expect to work like overloaded operators, and things like macros that are just bugs we should fix).
 - the comment about ObjC that is pretty unclear (I think it's just saying "there might be bugs", which is true of everything)
 - generated files exclusion (I don't think it's true?)
 - 50 files, this is very easy to observe interactively